lib/hotwire/livereload/version.rb

```diff
@@ -1,5 +1,5 @@
 module Hotwire
   module Livereload
-    VERSION = "1.1.0"
+    VERSION = "1.2.0"
```

Comment:
The version will be bumped automatically during release.
```suggestion
    VERSION = "1.1.0"
```